### ISSUE TITLE
docker-py build does not tag the image with the input argument

### ISSUE THREAD

**Issue by huikang** [docker/docker-py#1325]
I use docker-py to build an image; the code looks like
```
                                build(path=image.path,
                                          tag="myimage",
                                          buildargs=buildargs):
```

However, after the script completes with successfully built message, docker image show
```
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
<none>              <none>              db3bf7f68889        1 seconds ago       128.2 MB
```

Looks like the tag parameter does not take effect. Any suggestions? Thanks.

**Comment by huikang** [comment]
I think I found the root cause is the recent released pip package **requests 2.12.2**. After rolling back to **2.12.1**, everything works fine.

**Comment by jeffrey4l** [comment]
This is duplicated with #1321
